test/integration/pack/test_pack.py

```diff
@@ -27,21 +27,26 @@ def assert_bundle_structure(path):
     assert os.path.isdir(os.path.join(path, "var/run"))
     assert os.path.isdir(os.path.join(path, "var/lib"))
     assert os.path.isdir(os.path.join(path, "var/log"))
-    assert os.path.isdir(os.path.join(path, "env/bin"))
-    assert os.path.isdir(os.path.join(path, "env/modules"))
+    assert os.path.isdir(os.path.join(path, "bin"))
+    assert os.path.isdir(os.path.join(path, "modules"))
 
 
-def assert_env(path):
+def assert_env(path, artifacts_in_separated_dirs):
     with open(os.path.join(path, config_name)) as f:
         data = yaml.load(f, Loader=yaml.SafeLoader)
-        assert data["tt"]["app"]["instances_enabled"] == "instances_enabled"
-        assert data["tt"]["app"]["wal_dir"] == "var/lib"
-        assert data["tt"]["app"]["vinyl_dir"] == "var/lib"
-        assert data["tt"]["app"]["memtx_dir"] == "var/lib"
-        assert data["tt"]["app"]["bin_dir"] == "env/bin"
+        assert data["tt"]["app"]["instances_enabled"] == "instances.enabled"
+        if artifacts_in_separated_dirs:
+            assert data["tt"]["app"]["wal_dir"] == "var/wal"
+            assert data["tt"]["app"]["vinyl_dir"] == "var/vinyl"
```

Comment:
Do we have a test where the only one dir differs from others? If no, please add it.